backends/batch/replays.go

```diff
@@ -90,6 +92,40 @@ func (b *Batch) listReplays() ([]ReplayOutput, error) {
 	return output, nil
 }
 
+// ArchiveReplay archives a replay
+func (b *Batch) ArchiveReplay() error {
+	if err := b.archiveReplay(); err != nil {
+		return err
+	}
+
+	b.Printer("Successfully archived replay")
+
+	return nil
+}
+
+func (b *Batch) archiveReplay() error {
+	res, code, err := b.Delete("/v1/replay/" + b.Opts.Batch.ReplayID)
+	if err != nil {
+		return errors.Wrap(err, errReplayArchiveFailed.Error())
+	}
+
+	if code > 299 {
+		errResponse := &BlunderErrorResponse{}
+		if err := json.Unmarshal(res, errResponse); err != nil {
+			return errReplayArchiveFailed
+		}
+
+		for _, e := range errResponse.Errors {
+			err := fmt.Errorf("%s: %s", errReplayArchiveFailed, e.Message)
+			b.Log.Error(err)
+		}
+
+		return err
```

Comment:
does this just return `nil` in all cases? This should be returning a `Non-2xx Error`.

Reply:
Yeah good catch, returning an actual error now